health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -0,0 +1,174 @@
+            {
+                body: updatedPayload
+            },
+            {
+                onSuccess: (data) => {
+                    setShowToast({ key: "success", label: t("WORKFLOW_UPDATE_SUCCESS") });
+                    onSuccess && onSuccess(data); // Call the onSuccess callback if provided
+                    onClose();  // Close popup after success
+                },
+                onError: (error) => {
+                    setShowToast({ key: "error", label: t("ERROR_WHILE_UPDATING_WORKFLOW") });
+                    onError && onError(error); // Call the onError callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)